streams/src/test/java/org/apache/kafka/streams/state/internals/WindowKeySchemaTest.java

```diff
@@ -148,22 +169,81 @@ public void testLowerBoundMatchesTrailingZeros() {
         assertThat(
             "appending zeros to key should still be in range",
             lower.compareTo(
-                WindowStoreUtils.toBinaryKey(
+                    WindowKeySchema.toStoreKeyBinary(
                         new byte[]{0xA, 0xB, 0xC, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
                         Long.MAX_VALUE - 1,
                         0
                 )
             ) < 0
         );
 
-        assertThat(lower, equalTo(WindowStoreUtils.toBinaryKey(new byte[]{0xA, 0xB, 0xC}, 0, 0)));
+        assertThat(lower, equalTo(WindowKeySchema.toStoreKeyBinary(new byte[]{0xA, 0xB, 0xC}, 0, 0)));
     }
-    
-    private List<Integer> getValues(final HasNextCondition hasNextCondition) {
-        final List<Integer> results = new ArrayList<>();
-        while (hasNextCondition.hasNext(iterator)) {
-            results.add(iterator.next().value);
-        }
-        return results;
+
+    @Test
+    public void shouldSerializeDeserialize() {
```

Comment:
shouldn't this test be in a different class? same next 3 tests below

Reply:
These tests involves both a serializer and a deserializer, so I think this is a better place to be in than `XXWindowedSerializer/DeserializerTest`. Actually I should bring `testWindowDeserializeExpectedWindowSize` to here as well.

Reply:
See my comment about. Those classes are defined in `WindowedSerde` and should be part of a `WindowedSerdeTest` class.